tests/cases/standalone/insert/insert_default.result

```diff
@@ -0,0 +1,66 @@
+CREATE TABLE test1 (i INTEGER, j BIGINT TIME INDEX, k STRING DEFAULT 'blabla');
+
+Affected Rows: 0
+
+INSERT INTO test1 VALUES (DEFAULT);
+
+Error: 1004(InvalidArguments), Columns and values number mismatch, columns: 3, values: 1
+
+INSERT INTO test1 VALUES (DEFAULT, DEFAULT, DEFAULT);
+
+Error: 1004(InvalidArguments), No valid default value can be build automatically, column: j
```

Comment:
Should be `can be built automatically` here ? It's a typo I think.

Could you please also fix the typo in code. It's my bad, sorry about that.

Reply:
`build` -> `built`.

Reply:
haha nice catch